lib/rules/selector-max-compound-selectors/index.mjs

```diff
@@ -51,7 +65,11 @@ const rule = (primary) => {
 				// Compound selectors are separated by combinators, so increase count when meeting one
 				if (childNode.type === 'combinator') {
 					compoundCount++;
+				} else if (optionsMatches(secondaryOptions, 'ignoreSelectors', childNode.toString())) {
+					compoundCount -= previousNodeWasCompoundSeparator ? 1 : 0;
```

Comment:
stylistic

```suggestion
					compoundCount -= Number(previousNodeWasCompoundSeparator);
```

Reply:
I prefer the current code because `Number()` is slightly less explicit.